src/types/Patterns.ts

```diff
@@ -82,6 +82,7 @@ export const Patterns = {
   connectors: {
     NAME: /^[^-\s()'"#@.&](?!.*[%&?,';:!\s-]{2}).{1,19}$/,
     COUNTRY: /^[A-Z]{2}$/,
+    CUSTOMER_LINK: /^.*$/,
```

Comment:
this does match ANY string and therefore checks for this pattern are obsolete. If this should match any URL - check the already existing function `isURL(...)`

Reply:
I wanted to accept any string, so had added it. Now removed. Pattern is not mandatory for the ConnectorForm component